views/install.sh.erb

```diff
@@ -26,6 +26,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+download_timeout=10
```

Comment:
As someone currently on Australian DSL this needs to be a LOT higher as a default.